apps/web/screens/UniversitySearch/UniversitySearch.tsx

```diff
@@ -0,0 +1,1210 @@
+                                    nameIs:
+                                      locale === 'en'
+                                        ? dataItem.nameEn
+                                        : dataItem.nameIs,
+                                    iconSrc: universities.filter(
+                                      (x) => x.id === dataItem.universityId,
+                                    )[0].logoUrl,
+                                  })
+                                }
+                                checked={
+                                  selectedComparison.filter(
+                                    (x) => x.id === dataItem.id,
+                                  ).length > 0
+                                }
+                                id={dataItem.id}
+                              />
+                            }
+                            sidePanelConfig={{
+                              cta: createPrimaryCTA(dataItem),
+                              buttonLabel: n('apply', 'Sækja um'),
+                              items: [
+                                {
+                                  icon: (
+                                    <Icon
+                                      icon={'school'}
+                                      type="outline"
+                                      color="blue400"
+                                    />
+                                  ),
+                                  title: `${n(
+                                    dataItem.degreeType,
+                                    TranslationDefaults[dataItem.degreeType],
+                                  )}`,
+                                },
+                                {
+                                  icon: (
+                                    <Icon
+                                      icon={'calendar'}
+                                      type="outline"
+                                      color="blue400"
+                                    />
+                                  ),
+                                  title: `${n('begins', 'Hefst')} ${n(
+                                    dataItem.startingSemesterSeason,
+                                    TranslationDefaults[
+                                      dataItem.startingSemesterSeason
+                                    ],
+                                  )} ${dataItem.startingSemesterYear}`,
+                                },
+                                // {
+                                //   icon: (
+                                //     <Icon
+                                //       icon={'document'}
+                                //       type="outline"
+                                //       color="blue400"
+                                //     />
+                                //   ),
+                                //   title: `${n('entryExam', 'Inntökupróf')}: TODO`,
+                                // },
```

Comment:
same as above

Reply:
This will be introduced into the code soon, just need data from universities

Reply:
I think that this should rather be entirely introduced once the data is available, instead of having this much code commented out all over the place. Im not sure, though, what the overall practice is for the web, what do you say @RunarVestmann?